exit_code_redirect.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Get the total number of arguments
+total_args=$#
```

Comment:
Can you add more comments to explain that this is working around a GitHub actions limitation with passing array inputs?

Reply:
Yep